index.html

```diff
@@ -1,279 +1,358 @@
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="works-button">See Project</button>
             </div>
-
-        </section>
-        <section class="form" id="contact">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form action="https://formspree.io/f/mnqwdbwy" method="post">
-                <div class="form-inputs">
-                    <div class="firstLast">
-                        <input type="text" placeholder="First Name" class="firstName"><input type="text"
-                            placeholder="Last Name" class="lastName">
-                    </div>
-                    <input class="fullName" type="text" id="name" placeholder="Full name" name="user_name" required
-                        maxlength="30">
-                    <input type="email" id="email" placeholder="Email address" name="user_email" required>
-                    <textarea id="message" name="user_message" required maxlength="500"
-                        placeholder="Enter text here"></textarea>
-                    <button class="button" type="submit" id="submit">Get in touch</button>
-                </div>
-            </form>
-        </section>
-
-        <footer>
-            <div class="footer-socials">
-                <ul>
-                    <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                    <li><a href=""><i class="fa-brands fa-angellist"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><img src="pics/medium.png" alt="medium"></a></li>
-                </ul>
+          </div>
+
+          <div class="works work5">
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="works-button">See Project</button>
+            </div>
+          </div>
+
+          <div class="works work6">
+            <div class="work-details">
+              <h3>Multi-Post Stories Gain+Glory</h3>
+              <ul class="language-tags">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="works-button">See Project</button>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Kindly as the requirements say, you have to delete all the 6 cards from your HTML file and create them dynamically using Javascript when the page loads. 
Your requirements: `You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.`

Reply:
- :bow: Thank you for the review.
- I created the cards dynamically on javascript to function such that when the see projects button is clicked , the card loads.